src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -290,7 +292,9 @@ public boolean configure(final StaplerRequest req, final JSONObject formData) th
                     this.getTargetApiURL(), this.getTargetApiKey(), this.getTargetHost(), this.getTargetPort());
 
         } catch(Exception e){
-            logger.warning("Unexpected exception occurred - " + e.getMessage());
+            StringWriter sw = new StringWriter();
+            e.printStackTrace(new PrintWriter(sw));
+            logger.severe("Unexpected exception occurred - " + sw.toString());
```

Comment:
Not sure I fully get this, is what is `sw.toString` show up as here?

Reply:
It will print out the content of the writer - the whole stack trace.

Reply:
Isn't the whole stack trace also printed when `e.printStackTrace(new PrintWriter(sw));` is run? Would we get two stacktraces printed?

Reply:
no and we don't want to print in stdout, we want to use the logger.